templates/taxi/car_detail.html

```diff
@@ -12,7 +12,31 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <br>
+  {% if user.id in driver_ids %}
+      <form action="" method="post">
```

Comment:
Consider specifying the `action` attribute explicitly, even if it's just the current URL, for clarity and maintainability.